stdlib/pervasives.ml

```diff
@@ -369,7 +373,7 @@ external seek_out : out_channel -> int -> unit = "caml_ml_seek_out"
 external pos_out : out_channel -> int = "caml_ml_pos_out"
 external out_channel_length : out_channel -> int = "caml_ml_channel_size"
 external close_out_channel : out_channel -> unit = "caml_ml_close_channel"
-let close_out oc = flush oc; close_out_channel oc
+let close_out oc = (try flush oc with _ -> ()); close_out_channel oc
```

Comment:
Buffered data could be silently lost if `flush` fails.  That makes this particular change seem dangerous.